compiler/rustc_target/src/spec/xtensa_base.rs

```diff
@@ -0,0 +1,29 @@
+use crate::spec::{LinkerFlavor, PanicStrategy, RelocModel, TargetOptions};
+use crate::{abi::Endian, spec::abi::Abi};
+
+pub fn opts() -> TargetOptions {
+    TargetOptions {
+        os: "none".to_string(),
+        env: String::new(),
+        vendor: String::new(),
+        endian: Endian::Little,
+        c_int_width: "32".to_string(),
+        linker_flavor: LinkerFlavor::Gcc,
+        executables: true,
+        max_atomic_width: Some(32),
```

Comment:
Remove this from here

Reply:
All four lines or just max_atomic_width?

Reply:
Just `max_atomic_width: Some(32)`.